src/constants/locConstants.ts

```diff
@@ -640,6 +640,8 @@ export let queryFailed = l10n.t("Query failed");
 export let parameters = l10n.t("Parameters");
 export let loading = l10n.t("Loading");
 export let executionPlan = l10n.t("Execution Plan");
+export let executionPlanFileFilter = l10n.t("SQL Plan Files");
+export let executionPlanFileBaseName = l10n.t("plan");
```

Comment:
should this be localized?

Reply:
do we usually localize file name related things?

Reply:
I think there isn't a need to localize the basename or file extension, but we do want to localize "SQL Plan Files" since that's meant to be a human-readable string to describe the extension.